SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,66 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2025-02-13T10:10:09.000Z'
+  last-updated: '2024-02-13'
+  last-reviewed: '2024-02-13'
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-url: https://github.com/ossf/S2C2F-attestation-schema-and-tool
+  project-release: '0.0.0'
```

Comment:
Is there a versioning/strategy guidance for this project on OpenSSF more generally? I know this may be a placeholder, but does this three-part version structure imply [Semantic Versioning](semver.org)?

Reply:
Yes, the 0.0.0 is a placeholder until we have a working prototype. But yes, I would prefer us to adhere to Semantic Versioning :)

Reply:
It seems I cannot resolve comments for myself on this project yet so you can resolve this one for me, you answered my question. :smile: